geometry.go

```diff
@@ -92,7 +92,7 @@ func NewCollectionGeometry(geometries ...*Geometry) *Geometry {
 
 // MarshalJSON converts the geometry object into the correct JSON.
 // This fulfills the json.Marshaler interface.
-func (g *Geometry) MarshalJSON() ([]byte, error) {
+func (g Geometry) MarshalJSON() ([]byte, error) {
 	// defining a struct here lets us define the order of the JSON elements.
 	type geometry struct {
```

Comment:
We can't do that here because mapping one of the specific geometry types into an interface so it gets marshalled correctly.

Reply:
Sorry, probably because this isn't my native tongue but I don't understand what you mean.

Reply:
Just saying that my above comment for Features doesn't apply for Geometry because here we need to map one of 
```
	Point           []float64
	MultiPoint      [][]float64
	LineString      [][]float64
	MultiLineString [][][]float64
	Polygon         [][][]float64
	MultiPolygon    [][][][]float64
```
into just a `geometry` attribute in the json.

Reply:
I get it… good.